Bottom: d87179a562b12480ad771842aeb0c0087067e110
Top:    c4d683f16a33e298b95f116e50af963a90bccdab
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 17:02:02 +0100

Win32: Select the correct GL libraries to link against


---

diff --git a/configure.ac b/configure.ac
index 74d6e9e..cd696a2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -429,7 +429,7 @@ Either do not use --enable-gl or enable the gtk HID.])
 	esac
 
 	GLU_CFLAGS=
-	GLU_LIBS=-lGLU
+	GLU_LIBS=-lglu32 -lopengl32
 
 	AC_DEFINE([ENABLE_GL], 1,
 		[Define to 1 if GL support is to be compiled in])
